app/views/early_bird_dashboard/index.html.slim

```diff
@@ -62,3 +62,4 @@
                 br
               - else
                 span.constituency #{question.member_constituency}
+            p =link_to('Email Parli-branch about this question', "mailto:#{Settings.mail_reply_to}?subject=Question #{question.uin}")
```

Comment:
I don't have much experience of slim, but it seems to be standard to have a space between `=` and the content.
You might also want to put the link_to on a separate line.
```suggestion
            p 
              = link_to('Email Parli-branch about this question', "mailto:#{Settings.mail_reply_to}?subject=Question #{question.uin}")
```

Reply:
Hi Andrew,
Thank you for your feedback.
Previous developer's standard way of writing slim was to have everything on one line and to change only this P would look odd

Reply:
Fair enough, consistency is good!